apps/aecore/test/pow_sha256_tests.erl

```diff
@@ -0,0 +1,83 @@
+%%%=============================================================================
+%%% @copyright (C) 2017, Aeternity Anstalt
+%%% @doc
+%%%   Unit tests for the pow_sha256 module
```

Comment:
Do we plan to write `@doc`s for all modules? To me these are unnecessary noice, but that's only my personal opinion.

@wagerlabs @sennui ?

Reply:
Let's drop the docs for now and get to them once the code has settled.